### ISSUE TITLE
Bug486584Test.testFullBuildWhenClasspathChanged_3 is flaky

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-eclipse#606]
Bug486584Test.testFullBuildWhenClasspathChanged_3 is flaky

```
Fehlermeldung

no fullbuild sheduled expected:<0> but was:<1>

Stacktrace

java.lang.AssertionError: no fullbuild sheduled expected:<0> but was:<1>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.eclipse.xtext.builder.impl.Bug486584Test.testFullBuildWhenClasspathChanged_3(Bug486584Test.java:156)

```

**Comment by cdietrich** [comment]
can be reproduced by running the test in a loop

```
while mvn clean install -Dtest=Bug486584Test; do :; done
```

**Comment by cdietrich** [comment]
still an issue

**Comment by ArneDeutsch** [comment]
Seems the test cases are order sensitive. If I run `testFullBuildWhenClasspathChanged` alone it fails each time.

**Comment by cdietrich** [comment]
i can confirm that. this looks even more strange to me

**Comment by cdietrich** [comment]
@szarnekow. i ignored the test for now. would be nice if you could give us some ideas/hints

**Comment by cdietrich** [comment]
Fixed in 2.17
